models/topicsModels.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+const fetchTopics = () => {
+  return db.query(`SELECT * FROM topics`).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "not found" });
```

Comment:
We don't need a Promise.reject here. The only thing that can go wrong at this point is if someone puts an incorrect endpoint which is being handled separately in your `app.use(*)`